databases/week2/p2_school_database.sql

```diff
@@ -0,0 +1,25 @@
+CREATE DATABASE SCHOOL_DATABASE
```

Comment:
Good job also here!

We do like consistency, so it would be nice with either lower- or upper case names (we also have the database hyf_lesson2)

Reply:
Thanks! I'll notice that. I just didn't know which format would be better to use.